### ISSUE TITLE
Windows 10 ;__;

### ISSUE THREAD

**Issue by SSCBryce** [qTox/qTox#1072]
I know this has to be the most backburner problem ever, and I understand that don't worry, but I wanted to say that recently it stopped working on Windows 10 (9926) even though it was working in the beginning.

![jchmkw](https://cloud.githubusercontent.com/assets/2372953/5986822/ae330f24-a8d9-11e4-84ff-a4775d484b70.png)

Gomen! And sank-yuu na!

**Comment by tux3** [comment]
There was an issue with code-signing, it should be fixed now.
Thanks for reporting it.

**Comment by Kanrinin-sama** [comment]
I don't know if posting here bumps this or if I have to make a new Issue, but the problem persists. Only this time I can't figure out how to trigger it. It just seems like, if I leave it on long enough, and try to restart it when it has no update, it will start using the same error as before until I reinstall it. I've tried both a full wipe, and just installing over with the latest binary. Just going to post this for now until I think I'm on to something else.

**Comment by tux3** [comment]
@Farfie can you upload a copy of the files that fail to start ?

**Comment by Kanrinin-sama** [comment]
Right now, it works, because I reinstalled (again) sometime yesterday. But every release since you said you fixed it has stopped working eventually, until I download the latest binary and reinstall.

But you did make progress. Because I went from not being able to turn it on at all, to this error appearing eventually, and fixable by reinstalling. The question of course, is, what is breaking it?

And I have no lead on this. Well, I have one lead.. maybe if you push an update while my qTox is running, then I restart to check for updates... it errors when I try to start it again. And continues to error until I download the latest binary and install myself? That is the only thing I can think of that might do it.

**Comment by tux3** [comment]
Post the qtox.log please, it's in C:\Users\You\AppData\Local\tox\qtox.log

**Comment by Kanrinin-sama** [comment]
Didn't find it there, but in Roaming.
http://pastebin.com/0rZY9ZrL

**Comment by tux3** [comment]
Do you still have this folder C:/Users/Farfie/AppData/Roaming/tox/update ?
If so, delete it. I'll be doing some tests on my end in the meantime.

**Comment by Kanrinin-sama** [comment]
Sorry, neet so schedule has me sleeping all day x.x
And no, no such folder exists.

Wish I could be more helpful, but this bug seems to be so random. I'll try restarting it every time I can remember throughout the night, and see if it happens, and post here as soon as it does. But it feels like at this rate, it would have to be something external causing it, like the updater expecting to install an update that's not downloaded, but thinks it exists locally already. I'm a moron so I haven't tried running the regular binary when this has been going on yet, but I'll do it next time it breaks...

**Comment by cjpit** [comment]
I'm getting the same issues on windows 8 - has been a problem for a while , so i downloaded the latest ver, and reinstalled.  update came through, tried to update, and got the same issue with the 7b

**Comment by Kanrinin-sama** [comment]
Alright so it happened again finally, and the only thing I noticed is that I got that update last night, and when I let it go through, it restarted fine and everything was dandy. I think I even restarted it again just to see if it would boot up, and it worked. I should really write this stuff down or something...

Anyway, I turned the computer off for the night, and when I woke up and turned the computer on, qTox is throwing the same error again when I try to run it. It's the same error for updater, or the binary itself. Since I have restarted multiple times without updating the last few days, and it worked just fine, there must be something about the entire path of update -> restart machine -> error.

The log just says "About to start the qtox updater to install a local update" multiple times, assuming because I tried to run it multiple times when it was throwing the error.

**Comment by tux3** [comment]
Maybe it's related to the updater not dropping privilege and running qTox as admin ?

**Comment by Kanrinin-sama** [comment]
You're saying running as admin would fix the problem, or it needs to not run as admin? Would a restart affect these choices the software is making/failing to make?

**Comment by Kanrinin-sama** [comment]
Just woke up, and got the same case today.
Restarted computer last night, turned qtox back on, it had an update.
Clicked yes, restarted it, it came back on.
Even restarted it a few times for good measure.
But the only difference between then and now is that the whole machine has been restarted, and now the error comes up. The only fix that I know is to reinstall. I do it by just downloading the newest binary and running it, I don't uninstall first.

One peculiar thing about this though is that when I do reinstall like this, and launch it for the first time, is that the updater comes up. But why should the updater come up if I just installed the newest binary? Perhaps the update last night never actually happens? Because that update window never comes up during the part where I'm updating before the computer restart, the program just restarts. Not until the next day, when the program fails to load and I reinstall with the binary, does the little qtox updater window come up and finish.

I think it's perhaps time you added a version information section in some kind of about tab for qtox? It's obviously nice to have in general, and for this bug, I would be able to see if after I click yes to updating, and restart, it /actually/ updated. At this point I would say that the updater is not actually updating, but something is preventing it from throwing the above error, until the whole OS is restarted, in which then it begins to error relentlessly (trying to load the updater or the binary, and as admin or not, makes no change).
